.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,6 +10,7 @@ See also #23
 
 ## Checklist
 
+- [ ] Agreed to the CLA (Contributor License Agreement) by clicking on this [link](https://cla.strongloop.com/agreements/strongloop/loopback-next) and completing it
```

Comment:
Maybe move it to the bottom of the list. The text can be:

- [ ] Agree to the CLA (Contributor License Agreement) by [clicking and signing](https://cla.strongloop.com/agreements/strongloop/loopback-next)

Reply:
+1

Reply:
@raymondfeng , addressed your comment. :)